CDP4CrossViewEditor/OfficeRibbon/CrossViewEditorRibbonPart.cs

```diff
@@ -223,12 +223,6 @@ public override async Task OnAction(string ribbonControlId, string ribbonControl
         /// </param>
         private void LaunchCrossViewEditorAsync(string iterationId)
         {
-            if (iterationId == string.Empty)
```

Comment:
Do we enable the ribbon button only if an iteration is selected?

Reply:
R1: Yes.